android-core/src/androidTest/kotlin/com.mparticle/networking/MParticleBaseClientImplTest.kt

```diff
@@ -20,11 +23,166 @@ class MParticleBaseClientImplTest : BaseCleanInstallEachTest() {
+                    )
+                    .addDomainMapping(DomainMapping.eventsMapping(eventsUrl).build())
+                    .build()
+            )
+            .build()
+        MParticle.start(options)
+        val baseClientImpl = AccessUtils.getApiClient() as MParticleBaseClientImpl
+        val url =
+            baseClientImpl.getPodUrl(NetworkOptionsManager.MP_URL_PREFIX, prefix, false)
+        assertEquals("${NetworkOptionsManager.MP_URL_PREFIX}.mparticle.com", url)
+    }
+
+    @Test
+    fun testAllPrefixes() {
+        val mConfigManager = ConfigManager(
+            mContext,
+            MParticle.Environment.Production,
+            "some api key",
+            "some api secret",
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
+        )
+
+        // Following are the fake APIs for testing purposes.
+        val map = mapOf<String, String>(
+            Pair("us1-1vc4gbp24cdtx6e31s58icnymzy83f1uf", "us1"),
+            Pair("us2-v2p8lr3w2g90vtpaumbq21zy05cl50qm3", "us2"),
+            Pair("eu1-bkabfno0b8zpv5bwi2zm2mfa1kfml19al", "eu1"),
+            Pair("au1-iermuj83dbeoshm0n32f10feotclq6i4a", "au1"),
+            Pair("st1-k77ivhkbbqf4ce0s3y12zpcthyn1ixfyu", "st1"),
+            Pair("us3-w1y2y8yj8q58d5bx9u2dvtxzl4cpa7cuf", "us3"),
+            Pair("kajsdhasdiuyaiudiashhadjhdasjk", "us1")
+        )
+        val identityUrl = mRandomUtils.getAlphaString(20)
+        val configUrl = mRandomUtils.getAlphaString(20)
+        val audienceUrl = mRandomUtils.getAlphaString(20)
+        val eventsUrl = mRandomUtils.getAlphaString(20)
+        val options = MParticleOptions.builder(mContext)
+            .credentials(apiKey, "secret")
+            .networkOptions(
+                NetworkOptions.builder()
+                    .addDomainMapping(
+                        DomainMapping.audienceMapping(audienceUrl)
+                            .build()
+                    )
+                    .addDomainMapping(
+                        DomainMapping.configMapping(configUrl)
+                            .build()
+                    )
+                    .addDomainMapping(
+                        DomainMapping.identityMapping(identityUrl)
+                            .build()
+                    )
+                    .addDomainMapping(DomainMapping.eventsMapping(eventsUrl).build())
+                    .build()
+            )
+            .build()
+        MParticle.start(options)
+        val baseClientImpl = AccessUtils.getApiClient() as MParticleBaseClientImpl
+        map.forEach { key, value ->
+            mConfigManager?.setCredentials(key, value)
+            val prefix = mConfigManager?.podPrefix
+            assertEquals(value, prefix)
+            assertEquals(
+                "${NetworkOptionsManager.MP_URL_PREFIX}.$prefix.mparticle.com",
+                baseClientImpl.getPodUrl(NetworkOptionsManager.MP_URL_PREFIX, prefix, true)
```

Comment:
nit:  

Can you update the formatting of this assert to match the formatting of lines 160-164, 168-172, and 176-180?